src/rm_cms.c

```diff
@@ -0,0 +1,328 @@
+#include <strings.h>        // strncasecmp
+#include <stdlib.h>         // malloc
+#include <math.h>           // ceil, log10f
+
+#include "version.h"
+#include "rmutil/util.h"
+
+#include "rm_cms.h"
+#include "cms.h"
+
+RedisModuleType *CMSketchType;
+
+typedef struct {
+    const char *key;
+    long long value;
+} CMSPair;
+
+static int CompareCStringToRedisString(char *str, RedisModuleString *redisStr) {
```

Comment:
Don't use CamelCase for functions that aren't objectish, use snakeCase. Also, I believe we have this exact function in rmutil, so you can borrow from there.

Also, I like shorter names. It gets difficult to read in code if you have giant names followed by other giant names.

e.g. `rstringCmp`